wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -24,16 +26,40 @@ namespace :wagn do
 
     # Note Cardio based, but some use cases need this in app dir.  Solution?
     ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
-     
+
     puts "creating"
     Rake::Task['db:create'].invoke
 
     puts "loading schema"
     Rake::Task['db:schema:load'].invoke
-    
+
+    Rake::Task['wagn:seed'].invoke
+  end
+
+  desc "clear and load fixtures with existing tables"
+  task :create do
+    ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
+
+    Rake::Task['wagn:clear'].invoke
+
+    Rake::Task['wagn:seed'].invoke
+  end
+
+  desc "empty the card tables"
+  task :clear => :environment do
+    conn = ActiveRecord::Base.connection
+
+    puts "delete all data in bootstrap tables"
+    WAGN_BOOTSTRAP_TABLES.each do |table|
+      conn.delete "delete from #{table}"
+    end
+  end
+
+  desc "Load bootstrap data into database"
+  task :seed do
```

Comment:
I thought (not totally sensibly) that adding cards to an existing app would mean that often we would not want to *start fresh by dropping and re-loading the schema*, but as I commented above that isn't quite right.

I was playing with adding couple of new subcommands and tasks.  The standard way, which is just like it does now, drop and reload, then seed.  The other way is clear data from all the card tables and then seed.

Let's see:
LOAD_TASKS= {
  'reseed' => 'create',
  'seed' => 'recreate',
  'load' => 'seed'
}
Sorry for that, it isn't pretty, but it will be ok when we finalize this stuff.  It maps the wagn subcommands to wagn: namespace tasks.  So, seed => recreate, which is what the old create did (that's part of what I'm apologizing for, it's not right, but I think I flipped them when I realized we didn't want the external commands changing (wagn X) because the are used in the build).

To me, recreate is what we do when we drop, reload schema and seed, and create should just create (isn't that would db:create does ?).  It is *wagn load* to just run rake wagn:seed.